docs/Plugins.rst

```diff
@@ -745,6 +745,42 @@ Note that Quickfort mode is only for compatibility with the legacy Python Quickf
 DFHack `quickfort` script does not need Quickfort mode to be enabled. The `quickfort` script
 will successfully integrate with buildingplan as long as the buildingplan plugin is enabled.
 
+Global settings
```

Comment:
This might be worth cross-referencing from the "Buildingplan integration" section of the quickfort user guide (I briefly assumed it was there already)

Reply:
done. I actually thought it was already there too..

Reply:
I was actually thinking of a link, but I can add that. Thanks for updating it!